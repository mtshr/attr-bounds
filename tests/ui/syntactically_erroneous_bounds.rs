use attr_bounds::bounds;

#[bounds(T Clone)]
struct Foo<T>(T);

fn main() {}
