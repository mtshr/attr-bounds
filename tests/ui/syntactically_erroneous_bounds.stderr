error: expected `:`
 --> tests/ui/syntactically_erroneous_bounds.rs:3:12
  |
3 | #[bounds(T Clone)]
  |            ^^^^^
